diff --git a/src/Wt/Http/Request b/src/Wt/Http/Request
index 3c63468..b588142 100644
--- a/src/Wt/Http/Request
+++ b/src/Wt/Http/Request
@@ -405,7 +405,7 @@ public:
 
    static void parseCookies(const std::string& cookie,
 			    std::map<std::string, std::string>& result);
-
+   ~Request();
 private:
   const WebRequest *request_;
   const ParameterMap& parameters_;
@@ -416,7 +416,7 @@ private:
 
   Request(const WebRequest& request, ResponseContinuation *continuation);
   Request(const ParameterMap& parameters, const UploadedFileMap& files);
-  ~Request();
+
 
   friend class Wt::WResource;
   friend class Wt::WebSession;
